content/en/docs/reference/glossary/endpoint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Endpoints
+id: endpoints
+date: 2020-04-23
+full_link: 
+short_description: >
+  Endpoints tracks the IP Addresses of the objects the service send traffic to.
+
+aka:
+tags:
+- fundamental
+---
+Endpoints tracks the IP addresses of a group of resources that are exposed using a {{< glossary_tooltip text="Service" term_id="service" >}}.
+
+<!--more-->
+
+Endpoints can be updated automatically through {{< glossary_tooltip text="labels" term_id="label" >}} and {{< glossary_tooltip text="selectors" term_id="selector" >}}, or you can manually configure your endpoints depending on your use case.
+Multiple Endpoints can be grouped together with {{< glossary_tooltip text="EndPointSlice" term_id="endpoint-slice" >}}.
```

Comment:
This isn't quite right. EndpointSlice resources improve on the Endpoints resource and its API:
- more scalable
- support for multiple address families

EndpointSlice and Endpoints both group together endpoints (the concept not the resource).